src/process_utility.c

```diff
@@ -336,8 +336,10 @@ process_truncate_chunk(Hypertable *ht, Oid chunk_relid, void *arg)
 #define TRUNCATE_RECURSE(rv) \
 	(rv)->inh
 #elif PG96
+#define SQL_INHERITANCE \
+    (strncmp(GetConfigOptionByName("SQL_inheritance", NULL, false), "on", sizeof("on")))
```

Comment:
Isn't `sizeof("on")` 3 (including `'\0'` char)? Why not replace with `2` instead?

Reply:
I can, wasn't sure was better going forward to maintain (e.g. if the string changed).

I did just realize I need a `== 0` though